Functions/InputOutput/xASL_io_DcmtkRead.m

```diff
@@ -239,19 +235,55 @@
 	end
 end
 
+if isfield(header, 'Private_0043_1083')
```

Comment:
Shouldn't we do the captured groups vs tokens in a separate issue from these here?